src/python/pants/backend/go/subsystems/gotest.py

```diff
@@ -72,6 +79,36 @@ class GoTestSubsystem(Subsystem):
         ),
     )
 
+    coverage_include_patterns = StrListOption(
```

Comment:
What should we name this option?

Should it have "import path" in the name?

Reply:
The current name seems fine to me!